freqtrade/optimize/backtesting.py

```diff
@@ -228,16 +228,18 @@ def _get_close_rate(self, sell_row: Tuple, trade: LocalTrade, sell: SellCheckTup
             # Special case: trailing triggers within same candle as trade opened. Assume most
             # pessimistic price movement, which is moving just enough to arm stoploss and
             # immediately going down to stop price.
-            if (sell.sell_type == SellType.TRAILING_STOP_LOSS and trade_dur == 0
-                    and self.strategy.trailing_stop_positive):
-                if self.strategy.trailing_only_offset_is_reached:
+            if sell.sell_type == SellType.TRAILING_STOP_LOSS and trade_dur == 0:
+                if not self.strategy.use_custom_stoploss and self.strategy.trailing_stop and \
+                   self.strategy.trailing_only_offset_is_reached and \
+                   self.strategy.trailing_stop_positive_offset is not None and \
```

Comment:
`trailing_stop_positive_offset` can't be None - json schema enforces it to be numeric and between 0 and 1 - so at max, it might not be explicitly set - in which case the default (set in `interface.py`) would apply.
(same for `trailing_only_offset_is_reached`)

I'm also not a particularily big fan of using `\` to split lines (call it bad experience from trailing characters when modifying a script on a server ...) - but would instead wrap the whole `if` into brackets - which allows newlines automatically.